core/scripts/local/CalculateContractBytecode.js

```diff
@@ -16,7 +16,7 @@ module.exports = async function(callback) {
   // Load contracts into script and output info.
   console.log("loading", argv.contract + ".json");
   let obj = require("./../../build/contracts/" + argv.contract + ".json");
-  const byteCodeSize = (obj.bytecode.length - 2) / 2;
+  const byteCodeSize = (obj.deployedBytecode.length - 2) / 2;
```

Comment:
After doing some more research into this we need need to be comparing to `deployedBytecode` rather than `bytecode` as `deployedBytecode` is what is actually pushed on-chain and thus this is what needs to be compared against. See: https://ethereum.stackexchange.com/questions/48534/how-to-estimate-my-contract-code-size-it-is-too-big-to-deploy-now